docs/conf.py

```
@@ -23,16 +23,14 @@
 
 # -- Path setup --------------------------------------------------------------
 
-import json
+# import json
 import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)